src/anm/events.js

```diff
@@ -164,28 +164,31 @@ MouseEventsSupport.prototype.dispatch = function(event) {
         return true;
     }
 
-    var dispatchedChild;  // found the matching child inside
-    if (owner.inside(localEvent)) {
-        owner.reverseEach(function(child) {
-            if (child.isActive()) {
-                dispatchedChild = child.dispatchMouseEvent(localEvent) ? child : null;
-                if (dispatchedChild) return false; // stop iteration of reverseEach
-            }
-        });
+    var dispatchedByOwner, // handled event myself
+        dispatchedByChild; // not handled myself, but found the matching child inside
 
-        if (!dispatchedChild) {
-            if (localEvent.type === 'mousemove') {
-                this.processMove(localEvent); // fire mouseenter/mouseexit if required
-            }
-            if ((localEvent.type === 'mouseclick') || (localEvent.type === 'mousedown')) {
-                state.pressedNode = this.owner;
-            }
-            this.owner.fire(localEvent.type, localEvent);
+    // here and below localEvent has properties `.x` and `.y`, so duck typing works
+    dispatchedByOwner = owner.inBounds && owner.inBounds(localEvent) && owner.inside && owner.inside(localEvent);
```

Comment:
> for example for image inBounds is not needed at all

for image inBounds checks if image is rotated and point belongs to its width/height

I changed this line and now it looks a bit better:

```javascript
dispatchedByOwner = !owner.isScene && owner.inside(localEvent);
```

`.inside` checks bounds first: https://github.com/Animatron/player/blob/develop/src/anm/animation/element.js#L1977, but does no checks for children, the reasoning for that (in case of dispatching events) I described above.

I'd like to merge it to see if it works "in real life" and if it's ok, fix code issues next week.